orchestration/src/main/java/com/sap/ai/sdk/orchestration/UserMessage.java

```diff
@@ -1,58 +1,48 @@
 package com.sap.ai.sdk.orchestration;
 
-import com.sap.ai.sdk.orchestration.model.MultiChatMessageContent;
 import java.util.List;
+import java.util.stream.Stream;
 import javax.annotation.Nonnull;
-import lombok.NoArgsConstructor;
 import lombok.Value;
 import lombok.experimental.Accessors;
 
 /** Represents a chat message as 'user' to the orchestration service. */
 @Value
 @Accessors(fluent = true)
-@NoArgsConstructor(force = true)
 public class UserMessage implements Message {
 
   /** The role of the assistant. */
   @Nonnull String role = "user";
 
   /** The content of the message. */
-  MessageContent content;
-
-  @Nonnull
-  @Override
-  public String content() {
-    return MessageContent.toString(content);
-  }
+  @Nonnull MessageContent content;
 
   @Override
   @Nonnull
-  public MessageContent getContent() {
-    return content != null ? content : new MessageContentSingle("");
+  public MessageContent content() {
+    return content;
   }
```

Comment:
(MInor)

You since we have fluent lombok accessors on this class, you can omit this method declaration. I expect it to be generated automatically.